### ISSUE TITLE
Add support for externalizing scripts into a files

### ISSUE THREAD

**Issue by bobtiernay-okta** [phillipuniverse/githook-maven-plugin#10]
This would be great for more elaborate hooks that don't fit comfortably inside a pom.

**Comment by phillipuniverse** [comment]
What did you have in mind? Like a reference to a bash script instead?

**Comment by bobtiernay-okta** [comment]
Right. That way you can reference it by file path. I'm not sure how the current config structure could be altered to be backwards compatible though.
